pkg/scheduler/apis/config/latest/latest.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
+
+import (
+	"k8s.io/component-base/config/v1alpha1"
+	"k8s.io/kube-scheduler/config/v1beta2"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config"
+	"k8s.io/kubernetes/pkg/scheduler/apis/config/scheme"
+)
+
+// GetInternalWithDefaults creates a default configuration of the latset versioned type.
+// This function needs to be updated whenever we bump the scheduler's component config version.
+//
+// Note: this function exists primarily for consumption by the cluster autoscaler, please
+// don't remove it even if it is not used by the main k8s pkg.
+func GetInternalWithDefaults() (*config.KubeSchedulerConfiguration, error) {
+	var versionedCfg v1beta2.KubeSchedulerConfiguration
+	versionedCfg.DebuggingConfiguration = *v1alpha1.NewRecommendedDebuggingConfiguration()
+
+	scheme.Scheme.Default(&versionedCfg)
```

Comment:
nit alternative: `scheme.Codecs.UniversalDecoder().Decode`. This does defaulting and conversion.

Reply:
we would actually end up with a longer logic and more error checking... lets keep it as is.